addons/l10n_in/static/src/helpers/hsn_summary.js

```diff
@@ -20,7 +20,7 @@ patch(accountTaxHelpers, {
             const quantity = base_line.quantity;
             const product_values = base_line.product_values;
             const uom = base_line.uom || {};
-            const taxes_data = base_line.taxes_data;
+            const taxes_data = base_line.taxes_data.taxes_data;
```

Comment:
Plz read the docstring -_- This code must remain consistent with the one python-side here:
https://github.com/odoo/odoo/blob/master/addons/l10n_in/models/account_tax.py#L49

So the current code is already correct and the input is wrong.

Reply:
@smetl My bad :sweat_smile: 
Done with the change.